app/service/file_svc.py

```diff
@@ -70,6 +72,13 @@ async def create_exfil_sub_directory(self, dir_name):
             os.makedirs(path)
         return path
 
+    async def create_exfil_operation_directory(self, dir_name):
+        now = datetime.now()
```

Comment:
Something seems fishy here. The timestamp will continually update, so you'll never have a "constant" path for all elements of an operation. I don't think that's the intended or desired behavior, so maybe we should look into finding a way to account for the progression of time?

Reply:
I see what you mean. If there are multiple files being exfiltrated in the operation, then multiple folders will be created. I was testing with compressed archives which masked this behavior. Will look into another way of classifying the operation.

Reply:
Could the operation ID not be used? If we wanted to stick with time we could also just use the start time of the operation. (`operation.id` or `operation.start` respectively)

Reply:
<img width="383" alt="image" src="https://user-images.githubusercontent.com/81784737/161653166-ca3e028c-b628-47a0-b70e-0260ad4c4488.png">

Code changes made so that operation name and timestamp in UTC are given. Due to how logger records timestamps, look a little messy. I could manipulate the datetime object if that improves readability.

Reply:
I'm not sure we have to display the timestamp exactly how it's formatted on the file - maybe we work out a wrapper to allow it to be displayed with a "date @ time" format? Not sure if that's worth the effort right now, though.

Reply:
Simple string substitution should suffice 

`x.start.strftime("%Y-%m-%dT%H:%M:%SZ").replace("T"," @ ")) `

<img width="360" alt="image" src="https://user-images.githubusercontent.com/81784737/162026660-7a9927da-17e7-40af-b0a2-dd41a7eb2e4a.png">

Reply:
I would personally stray away from having spaces in any path since that makes interacting with data more difficult. Additionally, the colons in the filename would break this feature on macOS and Windows since the colon is an illegal character in APFS, HFS, and NTFS. A replace for any character to an underscore might be a potential solution (though we also have to be careful with filename lengths).